xunit-autolabeler-v2/ast_parser/python/source_parsers/direct_invocation_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import ast
+import os
+
+from unittest.mock import MagicMock
+
+from . import direct_invocation
+
+
+def test_ignores_methods_with_existing_drift_attr():
+    fake_method = MagicMock()
+    fake_method.drift = '1234'
+
+    direct_invocation.parse([fake_method], 'my_class')
+
+    assert fake_method.drift == '1234'
```

Comment:
Do you also want to check the return value of `parse`?

Reply:
Done.